### ISSUE TITLE
ORCID icons are blocked

### ISSUE THREAD

**Issue by Bisaloo** [EFForg/privacybadger#2396]
#### What is your browser and browser version?

Firefox 67.0.2

#### What is broken and where?

Orcid icons are blocked (e.g. https://r-spatial.github.io/sf/)

#### What is the "culprit" domain?

members.orcid.org

#### What is your debug output for this domain?

```
**** ACTION_MAP for orcid.org
members.orcid.org {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1561346801597
}
orcid.org {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1559783022794
}
pub.orcid.org {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1553592003409
}
**** SNITCH_MAP for orcid.org
orcid.org [
  "mllg.github.io",
  "manuscriptcentral.com",
  "r4photobiology.info"
]
```

**Comment by ghostwords** [comment]
I think this is another Google Analytics-related problem: You first get first-party GA cookies and then you see them in third-party contexts (#367).

**Comment by ghostwords** [comment]
No other user error reports at this time.

**Comment by ghostwords** [comment]
Resolving as we haven't had other reports, and this shouldn't be a problem going forward now that Privacy Badger catches Google Analytics by default.

**Comment by Bisaloo** [comment]
@ghostwords, I'm able to reproduce this issue using the current master version of PB (8e8ad9838b74b6d13354163f78d362ca60dd44f9). Is it supposed to be fixed or am I missing something?

**Comment by ghostwords** [comment]
Ah I see there is another cookie, "X-Mapping-fjhppofk". It gets set for me by https://members.orcid.org/sites/default/files/vector_iD_icon.svg on https://r-spatial.github.io/sf/

**Comment by llrs** [comment]
I have also problems with one redirect towards ORCID website (not sure if related to this cookies). But the only solution I found was to disable Privacy Badger for a while.

**Comment by ghostwords** [comment]
Hi @llrs, I am not sure what you mean by "redirect towards ORCID website". Could you describe how to reproduce your issue?

**Comment by llrs** [comment]
Yes. You'll need an account on https://www.editorialmanager.com/binf/default.aspx
I was using Firefox on Linux and on this website you are required to link that account to ORCID. I simply couldn't with the privacyBadger activated.

I also found some problems on asocial login (but that's probably for another issue/thread)
